### PR TITLE
Fixed the SYN eater tests so they have enough privilege

### PR DESCRIPTION
**Pull request by knobunc**
These changes make the DROP_SYN_DURING_RESTART tests better match the
documented procedure.  I noticed we were getting errors in the docker
logs because the router was not set up to use root, so iptables was
failing.  This corrects that.

### CODE REVIEW

**Comment by knobunc** [comment]
@openshift/networking PTAL

**Comment by knobunc** [comment]
[test]

**Comment by danwinship** [comment]
> I noticed we were getting errors in the docker logs because the router was not set up to use root, so iptables was failing.

why was this not causing the tests to fail?

File: hack/util.sh [hunk 1]
```diff
@@ -229,7 +229,8 @@ function install_router() {
 	# Set the SYN eater to make router reloads more robust
 	if [[ -n "${DROP_SYN_DURING_RESTART:-}" ]]; then
 		# Rewrite the DC for the router to add the environment variable into the pod definition
-		echo "[INFO] Changing the router DC to drop SYN packets during a reload"
+        echo "[INFO] Changing the router DC to drop SYN packets during a reload"
```
Comment (knobunc) [inline hack/util.sh]: What's the preferred style?  The existing has 2 tabs, and the new has spaces.

**Comment by knobunc** [comment]
I'll see if I can add some tests that catch the failure explicitly later.  Originally the requested test was just to make sure it didn't break the router since getting a reload to stall long enough to drop packets is very unreliable and would be thought to flake too much.  Failing iptables used to be fatal, but then that was deemed too dangerous and that got made into a warning.

**Comment by knobunc** [comment]
Last failure was flake https://github.com/openshift/origin/issues/10773

**Comment by knobunc** [comment]
Flake https://github.com/openshift/origin/issues/11650 [test]

**Comment by knobunc** [comment]
[test]

**Comment by knobunc** [comment]
[test]

**Comment by knobunc** [comment]
[test]

**Comment by dcbw** [comment]
LGTM

**Comment by knobunc** [comment]
[merge]

**Pull request merged**